base/workflow/R/runModule.run.write.configs.R

```diff
@@ -5,14 +5,19 @@
 #' @return A modified settings object, invisibly
 #' @importFrom dplyr %>%
 #' @export
-runModule.run.write.configs <- function(settings, overwrite = TRUE) {
+runModule.run.write.configs <- function(settings, overwrite = TRUE,input_design=NULL) {
```

Comment:
spacing 
```suggestion
runModule.run.write.configs <- function(settings, overwrite = TRUE, input_design = NULL) {
```

(I won't comment on the rest of these individually, but please use the same spacing throughout the PR)